challenges-services/pom.xml

```diff
@@ -76,6 +76,10 @@
       <artifactId>commons-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.exoplatform.social</groupId>
+      <artifactId>social-component-service</artifactId>
+    </dependency>
```

Comment:
Could you place this dependency declaration to line 49 and add provided scope like above?
```xml
    <dependency>
      <groupId>org.exoplatform.social</groupId>
      <artifactId>social-component-service</artifactId>
      <scope>provided</scope>
    </dependency>
```

Reply:
done